src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -115,6 +115,8 @@ private static long GetEnumValue(bool isUnderlyingTypeUInt64, Enum enumVal, Cult
         /// <summary>
         /// Converts the given value object to the specified destination type.
         /// </summary>
+        [UnconditionalSuppressMessage("AotAnalysis", "IL3050:RequiresDynamicCode",
```

Comment:
The concern is whether the array of the enum is available. I do not see anything that guarantees it.

Reply:
Created an issue for this, #72140 and will remove this in the next commit